collectors/health_test.go

```diff
@@ -24,6 +25,24 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	healthChecksString = `{
```

Comment:
Does this need to be in yaml instead of json?

Reply:
oh, yes that does!